internal/migration/migration.go

```diff
@@ -0,0 +1,35 @@
+package migration
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/golang-migrate/migrate/v4"
+	"github.com/golang-migrate/migrate/v4/database"
+	_ "github.com/golang-migrate/migrate/v4/database/mongodb"
+	_ "github.com/golang-migrate/migrate/v4/source/file"
+)
+
+func Migrate(mongConnection string) error {
+	mongConnection = mongConnection + "/news"
+	migrationDriver, err := database.Open(mongConnection)
+	if err != nil {
+		return fmt.Errorf("failed to create instance %s", err)
+	}
+
+	defer func() {
+		if err := migrationDriver.Close(); err != nil {
+			log.Fatalf("failed to close migration driver %s", err)
+		}
+	}()
+
+	m, err := migrate.NewWithDatabaseInstance("file://migration", "news", migrationDriver)
```

Comment:
same question as in https://github.com/newUser1337/task-news/pull/1/files#r1474656554

Reply:
fixed